views/partials/beerpartial.hbs

```diff
@@ -0,0 +1,12 @@
+
+<div class = "beerPartial">
+    <div>
+    <a><img src={{this.image_url}} alt="beer" class = "image"></a>
+    </div>
```

Comment:
You are wrapping the image in a link but there is no href, so the images are not clickable yet.